Extension/DefaultOptionsExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Knp\Bundle\MenuBundle\Extension;
+
+use Knp\Menu\Factory\ExtensionInterface;
+use Knp\Menu\ItemInterface;
+
+/**
+ * An extension to dynamically configure default menu options.
```

Comment:
why "dynamically"? if the default options are in the configuration file, i would consider that rather static.